contracts/zero-ex/package.json

```diff
@@ -15,6 +15,7 @@
         "build:ts": "tsc -b",
         "pre_build": "run-s compile contracts:gen generate_contract_wrappers contracts:copy",
         "test": "yarn run_mocha",
+        "forge": "forge test -vvv",
```

Comment:
Let's not override the default forge command. There can be a test:integration or something like that.